packages/teleterm/src/ui/StatusBar/ShareFeedback/ShareFeedbackForm.tsx

```diff
@@ -39,9 +39,20 @@ export function ShareFeedbackForm({
         value={formValues.email}
         onChange={e => updateFormField('email', e.target.value)}
       />
+      <FieldInput
+        label="Company name (optional)"
+        autoFocus
```

Comment:
The company name field shouldn't have autofocus, should it?

Reply:
No, it shouldn't.